test/fork.js

```diff
@@ -59,3 +59,12 @@ test('exit after tests are finished', function (t) {
 			cleanupCompleted = event.completed;
 		});
 });
+
+test('fake timers do not break duration', function (t) {
+	fork(fixture('fake-timers.js'))
+		.then(function (info) {
+			var duration = info.tests[0].duration;
+			t.true(duration < 1000, duration + ' < 1000');
```

Comment:
Yep. I'm so tempted to go create tap support. We will self test someday, and then all will be right with the world.